main.cpp

```diff
@@ -1,6 +1,10 @@
 #include <iostream>
+#include "ScreenCapture.h"
+
+int main(int argc, char **argv) {
+    OutputCapture::ScreenCapture *capturer = OutputCapture::ScreenCapture::GetOrCreateInstance();
+
+    capturer->CaptureFullScreen("./screenshot.bmp");
```

Comment:
about your string "./screenshot.bmp", you should create an include file named constant and put a static constexpr char[] inside

Reply:
I totally agree but the main's function content is just here for demo purpose :D. I want the file name to be dynamic and be called in a different way later